bin/jam/build-for-npm.sh

```
@@ -1,76 +1,59 @@
+# Build the main entry point
+bun build --target=node $EXTERNALS \
+  --outfile "$DIST_FOLDER/index.js" \
+  "$ROOT/bin/jam/index.ts"
 
-# Fix un-compiled worker files to point to the ones we will compile manually.
-#
-# Despite using `-a` flag, @vercel/ncc does not bundle the worker files,
-# so they still point to some external files via `import` statements.
-# To fix that, we manually build workers and move the files inside.
+# Build workers separately (they run in worker_threads, referenced via new URL())
+bun build --target=node $EXTERNALS \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)